### DISCUSSION TITLE
ORC and Import Indicator in the web interface.

### DISCUSSION THREAD

**Discussion by noritaka97** [ciur/papermerge#410] [Ideas]
Hi everyone,

It's possible to have a visual indicator on the web interface to know if import or OCR is in progress ?

Thanks for your works.

**Comment by ciur** [comment]
@noritaka97, this is a feature being developed for next version (2.1). See [here an example](https://youtu.be/syRorezQasI?t=27) of it in action.

**Reply by noritaka97** [reply]
Thanks for you reply.
